### DISCUSSION TITLE
Reauthenticating with a new oauth token?

### DISCUSSION THREAD

**Discussion by slichtenthal** [homebridge-simplisafe3/homebridge-simplisafe3#260] [Q&A] [answered]
Hi,

    I am getting errors with the oauth token provided to the app, but the homebridge UI won’t allow me to enter a new oauth token in as it says Authentication has been setup already.  I tried to re-login using the console and homebridge-simplisafe3 login, but that is failing with an error initially trying to run xdg-open and then after I installed that, it is exiting with error 3.  

I also tried manually going to the URL listed in the login prompt and it is returning a 404 error. 

Any suggestions would be helpful!

Thanks!

**Comment by shamoon** [comment, marked as answer]
if authentication is already setup you should be able to click the button to show instructions which reveals the button to restart the process.
